test/LockTests.js

```diff
@@ -0,0 +1,195 @@
+/* eslint-env node, jasmine */
+import {
+  configure
+  // render,
+} from 'enzyme';
+
+import Adapter from 'enzyme-adapter-react-16';
+
+import { expect } from 'chai';
+
+// Utils
+import { mountHGComponent, removeHGComponent } from '../app/scripts/utils';
+
+configure({ adapter: new Adapter() });
+
+describe('Horizontal heatmaps', () => {
+  let hgc = null;
+  let div = null;
+
+  beforeAll(done => {
+    [div, hgc] = mountHGComponent(div, hgc, viewconf, done, {
+      style: 'width:800px; height:400px; background-color: lightgreen',
+      bounded: true
+    });
+  });
+
+  it('should export as SVG', () => {
```

Comment:
Is this testing locks or SVG export? It seems that the file name does not match the test contents.

Reply:
Yes, that's a good point. It actually does test the locks by virtue of loading the viewconf and not crashing but that requires some insider knowledge to realize. I added a couple tests to explicitly verify that the corrupt lock was removed.